packages/cli-tool/src/commands/generate/index.ts

```diff
@@ -52,11 +60,27 @@ export default class Generate extends Command {
       }
 
       this.setVersionControl(appName, answers.versionControl)
+      await this.setUIFramework(appName, answers.uiFramework)
+
+      // Clean files after all steps
+      this.cleanFiles(appName)
     } catch (error) {
       this.error(error as string | Error)
     }
   }
 
+  setUIFramework = async(appName: string, uiFramework: string): Promise<void> => {
+    if (uiFramework === 'bootstrap') {
```

Comment:
Maybe add something like this, since we have `tailwind` in options to select?
```ts
if (uiFramework === 'tailwind') {
    cli.info('Tailwind is not available yet')
    return
}
```

Reply:
nice catch! I initially wanted to handle both in the same PR, but we finally decided to have 2 separated PR so that it is made easier for reviews ;-) 

done in cfb7735 ✅